packages/gensx-openai/src/gsx-completion.tsx

```diff
@@ -108,7 +122,7 @@ export const ChatCompletion = gsx.StreamComponent<ChatCompletionProps>(
       return generateTokens();
     } else {
       const response = await gsx.execute<ChatCompletionOutput>(
-        <GSXChatCompletion {...props} stream={false} />,
+        gsxChatCompletionImpl({ ...props, stream: false }),
       );
```

Comment:
@EvanBoyle, @jmoseley - is this pattern acceptable? it felt strange to use gsx.execute with a function but it works fine and it was giving me issues without the execute

Reply:
Yes I think this is an ok way to skip the checkpoints for these. Perhaps we need a way to configure `execute` to not record a node?

What issues were you running into?

Reply:
I see this error if i don't use gsx.execute: 

Unexpected `await` of a non-Promise (non-"Thenable") value.eslint[@typescript-eslint/await-thenable](https://typescript-eslint.io/rules/await-thenable)

Perhaps it's really just a linting thing though? i never tried to run it

Reply:
oh yeah that is a linter issue. But using execute is ok.

Reply:
ahh i guess that's because i was using await with `gsxChatCompletionImpl` but it's not async.  but then when i remove the await and run it i do get a vague type error `TypeError: Cannot read properties of undefined (reading '0')`

Reply:
> oh yeah that is a linter issue. But using execute is ok.

Cool i'll keep it as is for now then